### PR TITLE
Add ability to add question

### PR DESCRIPTION
**Pull request by matmuchrapna**

### CODE REVIEW

File: levels/pages/faq/faq.jade [hunk 1]
```diff
@@ -335,3 +337,9 @@ block content
                 <div class='block__elem3'></div>
             </div>
             ```
+        a(id="ask-you-question")
+        .raw-text
+          :markdown
+            ## Not found the answer? Please ask your question, don't be shy.
```
Comment (varya) [inline levels/pages/faq/faq.jade]: Did not find the answer?

Comment (matmuchrapna) [inline levels/pages/faq/faq.jade]: updated entire pull request

File: levels/pages/faq/faq.jade [hunk 2]
```diff
@@ -335,3 +337,9 @@ block content
                 <div class='block__elem3'></div>
             </div>
             ```
+        a(id="ask-you-question")
+        .raw-text
+          :markdown
+            ## Did not found the answer? Please ask your question, don't be shy.
```
Comment (varya) [inline levels/pages/faq/faq.jade]: found -> find

Comment (matmuchrapna) [inline levels/pages/faq/faq.jade]: omg, my english in terrible state

**Pull request merged**